tendrl/monitoring_integration/grafana/webhook_receiver.py

```diff
@@ -1,19 +1,72 @@
-from gevent.pywsgi import WSGIServer
-
+import gevent
+import gevent.greenlet
+import json
 
-def application(env, start_response):
-    if env['PATH_INFO'] == '/grafana_callback':
-        start_response('200 OK', [('Content-Type', 'text/html')])
-
-        # TODO(Rishubh) Receive Data and call parsing functions with alert json from grafana
-        # data = env['wsgi.input'].read()
+from gevent.pywsgi import WSGIServer
+from gevent.server import StreamServer
+from tendrl.commons.event import Event
+from tendrl.commons.message import ExceptionMessage
+from tendrl.monitoring_integration.alert.handlers import AlertHandlerManager
 
-        return [b"<b>Recieved</b>"]
 
-    start_response('404 Not Found', [('Content-Type', 'text/html')])
-    return [b'<h1>Not Found</h1>']
+class WebhookReceiver(gevent.greenlet.Greenlet):
+    def __init__(self):
+        super(WebhookReceiver, self).__init__()
+        self.server = WSGIServer(
+            ('127.0.0.1', 8789),
```

Comment:
Arent these configurable? Or its always localhost? At least define constants.

Reply:
Localhost because it is assumed that monitoring integration will always be on same server as grafana.
Port is as per suggestion of Rohan
Yes constants should be defined instead of using it directly

Reply:
this is always localhost so no need to configure, i will define constant for it